.vscode/settings.json

```diff
@@ -1,3 +1,19 @@
 {
-    "phpclassgen.vendor": "TeamGantt\\Dues"
+    "phpclassgen.vendor": "TeamGantt\\Dues",
+    "workbench.colorCustomizations": {
```

Comment:
Excuse me?

Reply:
I'm removing the vscode settings in a follow-up commit

Reply:
live share patches the settings for live share colors and that could get super annoying to track 😱